core/chains/evm/client/simulated_backend_client.go

```diff
@@ -749,7 +749,7 @@ func (c *SimulatedBackendClient) ethGetLogs(ctx context.Context, result interfac
 		// lastTopic is the topic index of the last non-nil topic slice
 		//  We have to drop any nil values in the topics slice after that due to a quirk in FilterLogs(),
 		//  which will only use nil as a wildcard if there are non-nil values after it in the slice
-		for i := 0; i < lastTopic; i++ {
+		for i := 0; i <= lastTopic; i++ {
```

Comment:
Yes, although the method was only added to `SimulatedBackendClient` last week, to unblock work on this PR... so this will be the first time it's being called aside from tests.

Reply:
https://github.com/smartcontractkit/chainlink/pull/12360